readme.md

```diff
@@ -103,4 +104,47 @@ __*Note*__: `rejected` jobs will be filtered out and will disappear from the out
 [cron_doc]:docs/crontab/readme.md
 [conc_fut]:https://docs.python.org/dev/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor
 [thread]: https://docs.python.org/3.8/library/threading.html
-[delay_jp]:https://github.com/bunsenmurder/Notebooks/blob/master/jobFunnel/delay_algorithm.ipynb
\ No newline at end of file
+[delay_jp]:https://github.com/bunsenmurder/Notebooks/blob/master/jobFunnel/delay_algorithm.ipynb
+
+### easy_apply extension
+
+The easy_apply extension nicely integrates with JobFunnel to make your job search even easier!
+
+It will open as many browser tabs as you tell it to with randomly selected job links. It also marks those same jobs with a custom status of your choosing(the default is "archive")
+
+Once you have your scraped "master_list.csv" file, you can start using easy_apply.
+
+	cd JobFunnel/jobfunnel
+
+
+**Make sure that the "data" and "search" folders which contains the master_list are on this directory. Copy them to this directory if you have to. Make sure that there is a "settings.yaml" in "JobFunnel/jobfunnel" as well.**
+
+	./__main__.py easy_apply --ea_number 2
```

Comment:
Users shouldn't run main.py directly - this is the same as running “funnel easy apply...”